core/camel-main/pom.xml

```diff
@@ -102,6 +102,11 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Are you using this at all?

Reply:
yep I'm using it but I can replace it

Reply:
Okay since its not already widespread in use, lets try to keep the number of different JARs and ways to test to the same level

Reply:
done